homeassistant/components/cups/sensor.py

```diff
@@ -276,11 +276,11 @@ def device_state_attributes(self):
         if self._attributes is None:
             return None
 
-        high_level = self._attributes[self._printer]["marker-high-levels"]
+        high_level = self._attributes[self._printer].get("marker-high-levels", None)
```

Comment:
`None` is the default. 

```suggestion
        high_level = self._attributes[self._printer].get("marker-high-levels")
```

Reply:
Ah, I get it. Thanks.